src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculable {
```

Comment:
The class name 'Figure' should not be used for an interface that contains methods like 'getArea' and 'draw'. Consider splitting the behavior into two different interfaces since 'getArea' and 'draw' are conceptually different.